### PR TITLE
API-65: add integration tests on PQB [price]

### PR DESCRIPTION
**Pull request by skeleton**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | done
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/LocalizableFilterIntegration.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+    public function testOperatorNotEmpty()
+    {
+        $result = $this->execute([['a_localizable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => 'EUR'], ['locale' => 'en_US']]]);
+        $this->assert($result, ['product_one', 'product_two']);
+
+        $result = $this->execute([['a_localizable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => 'USD'], ['locale' => 'en_US']]]);
+        $this->assert($result, []);
+
+        $result = $this->execute([['a_localizable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => ''], ['locale' => 'en_US']]]);
```
Comment (ahocquard) [inline src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/LocalizableFilterIntegration.php]: currency and amount are completly ignored in the PriceFilter, you could add a test without those keys (for non empty and for empty).

File: src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/ScopableFilterIntegration.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        $result = $this->execute([['a_scopable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => ''], ['scope' => 'tablet']]]);
+        $this->assert($result, []);
+
+        $result = $this->execute([['a_scopable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => 'EUR'], ['scope' => 'tablet']]]);
+        $this->assert($result, ['product_two']);
+
+        $result = $this->execute([['a_scopable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => 'USD'], ['scope' => 'tablet']]]);
+        $this->assert($result, ['product_one']);
+
+        $result = $this->execute([['a_scopable_price', Operators::IS_NOT_EMPTY, ['amount' => '', 'currency' => 'USD'], ['scope' => 'ecommerce']]]);
```
Comment (ahocquard) [inline src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/ScopableFilterIntegration.php]: same comment : you could add a test without amount/currency

File: src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/LocalizableFilterIntegration.php (hunk 1 above)
Comment (skeleton) [inline src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/LocalizableFilterIntegration.php]: unfortunately, there is an error if keys are missing

Comment (skeleton) [inline src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Price/LocalizableFilterIntegration.php]: Ticket has been created by Laure (it concerns metric & price FYI)
